src/io/iter_sparse_prefetcher.h

```diff
@@ -126,15 +126,15 @@ class SparsePrefetcherIter : public PrefetcherIter {
   virtual bool Next(void) {
     return PrefetcherIter::Next();
   }
-  virtual const DataBatch &Value(void) const {
```

Comment:
I think both styles are fine: https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

No need to change this